src/components/entries/FEEL/FeelPopup.js

```diff
@@ -30,7 +30,7 @@ export const FEEL_POPUP_HEIGHT = 250;
 export default function FEELPopupRoot(props) {
   const {
     element,
-    eventBus = { fire() {}, on() {}, off() {} },
+    eventBus = { fire() { }, on() { }, off() { } },
```

Comment:
This looks absurd. I think we'd rather not have a rule which enforces space here.

<img width="1319" alt="image" src="https://github.com/user-attachments/assets/64c76f58-e231-46f7-a397-1dfb387f8963">

Reply:
That sneaked in from the moment I had the linter misconfigured. There is no such rule. I'll revert it.